packages/appstore/src/components/loader/devider/devider.scss

```
@@ -0,0 +1,11 @@
+.vertical-devider {
+    width: 1px;
+    height: 32px;
+    background-color: #f2f3f4;
```

let's use the css variables or scss variables that are available inside of deriv.app, 
```suggestion
    background-color: $COLOR_LIGHT_GRAY_6;
```
```suggestion
    background-color: var(--system-light-5-active-background, #d6dadb);
```